docs/05-PipelineLaunchOptions.md

```diff
@@ -101,3 +103,54 @@ Example for two entries/scans to be uploaded (one human subject and one phantom)
 ```
 
 
+## 5.2 - Pipeline Launch for BIDS datasets
+
+For now, the only way to run the pipeline is by running the `bids_import.py` 
+script manually via the terminal.
+
+To display the help section, run the following in the terminal:
+```bash
+bids_import.py -h
+```
+
+This will display the following help section:
+
+```bash
+usage  : bids_import -d <bids_directory> -p <profile> 
+
+options: 
+	-p, --profile        : name of the python database config file in dicom-archive/.loris-mri
+	-d, --directory      : BIDS directory to parse & insert into LORIS
+	-c, --createcandidate: to create BIDS candidates in LORIS (optional)
+	-s, --createsession  : to create BIDS sessions in LORIS (optional)
+	-v, --verbose        : be verbose
+```
+
+To run the BIDS import, simply run:
+```bash
+bids_import -d /PATH/TO/BIDS/TO/IMPORT -p database_config.py
+```
+
+If you wish to create candidates when running the import script, the `-c` 
+option needs to be added. To create sessions when running the import script, 
+the `-s` option need to be added as well.
+```bash
+bids_import -d /PATH/TO/BIDS/TO/IMPORT -p database_config.py -c -s
+```
+
+Finally, the verbose option can be turned on by using the option `-v` when 
+calling the script:
+```bash
+bids_import -d /PATH/TO/BIDS/TO/IMPORT -p database_config.py -v
+```
+
+
+**Note on the LORIS-MRI Python virtual environment**: in order to be able to 
+run `bids_import.py`, you need to be in the loris-mri Python virtual 
+environment. It should have been sourced when sourcing your LORIS-MRI 
+environment file. If this is not sourced, then simply run the following:
+```bash
+source /data/%PROJECT%/bin/mri/python_virtualenvs/loris-mri-python/bin/activate
+```
+To deactivate a Python virtual environment, simply type `deactivate` in the 
```

Comment:
Do users need to activate it every time they try to run the bids import?  Or if the server is rebooted? When should they deactivate ?

Reply:
When they source the environment file, it will automatically activate the virtual environment (unless they removed it from the environment file). But I think it is still good to know how you can activate the loris-mri Python virtual environment and deactivate it. Does that help?

Reply:
covers it, good to go.